docs/datastores.md

```diff
@@ -51,6 +51,17 @@ Uses [badger](https://github.com/dgraph-io/badger) as a key value store.
 }
 ```
 
+## pebbleds
+
+Uses [pebble](https://github.com/cockroachdb/pebble) as a key value store
+
+```json
+{
+	"type": "pebbleds",
+	"path": "<location of pebble inside repo>"
+}
+```
```

Comment:
@hsanjuan I know pebble is used by default in IPFS Cluster, and there is a lot of custom configuration in https://github.com/ipfs-cluster/ipfs-cluster/blob/v1.0.8/datastore/pebble/config.go#L25-L70

Two questions:
1. Does it make sense to expose them all via config here (as optional, use only when set)
2. For implicit defaults, should we use defaults from pebble, or ones from ipfs-cluster?